src/renderer/components/Wallet/WalletHeading/WalletHeadingPrimaryActions.vue

```diff
@@ -109,6 +110,17 @@ export default {
     }
   },
 
+  watch: {
+    // Never show the not-voting icon until knowing if the wallet is voting or not
+    currentWallet () {
```

Comment:
Watching this property may not be the best approach, as this reacts randomly. This will works better:

```suggestion
    'currentWallet.address' () {
```

Reply:
Thanks @luciorubeens